### PR TITLE
No limit for connection count from same Ip

### PR DESCRIPTION
**Pull request by predic8**

### CODE REVIEW

File: core/src/main/java/com/predic8/membrane/core/transport/Transport.java [hunk 1]
```diff
@@ -30,13 +30,18 @@
...
 	private Router router;
 	private boolean printStackTrace = false;
 	private boolean reverseDNS = true;
-	int concurrentConnectionLimitPerIp = 60;
+
+	/**
+	 * Number of allowed concurrent connections from one client ip
+	 * @default -1 No Limit
+	 */
+	private int concurrentConnectionLimitPerIp = -1;
```
Comment (rrayst) [inline core/src/main/java/com/predic8/membrane/core/transport/Transport.java:40]: This Javadoc comment will *not* be picked up by the annotation processor, as the target is not a setter method (but a field).

File: test/src/main/python/connections.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# Install: pip3 install locust
+# Run: locust -f connections.py
+#
+# See: https://docs.locust.io/en/stable/index.html
+class WebsiteUser(HttpUser):
+    # wait_time = between(5, 15)
+
+    @task
+    def fruit(self):
+        self.client.get("/shop/products/")
```
Comment (rrayst) [inline test/src/main/python/connections.py]: Was this file added by accident?

**Review by rrayst — CHANGES_REQUESTED** [review]

File: core/src/main/java/com/predic8/membrane/core/transport/Transport.java (hunk 1 above)
Comment (predic8) [reply core/src/main/java/com/predic8/membrane/core/transport/Transport.java:40]: Ok. I'll move the comment to the setter.

File: test/src/main/python/connections.py (hunk 2 above)
Comment (predic8) [reply test/src/main/python/connections.py]: No. But it is somewhat hidden. This is a locust script to provoke the limit from above. And it is useful for loadtesting. I'll move it and add some documentation.

**Review by rrayst — APPROVED** [review]

**Pull request merged**